Assignment7.txt

```diff
@@ -0,0 +1,19 @@
+Clarke Littlejohn
+
+1) if i recall correctly ++*P moves the pointer then gets the values, *p++ get  the value in the point the increments it
+   and *++p is not an apoerator that works.
+
+2) I am confused by what this si asking it will go by what it is prendence so to will go from right to left or left to right based on the highest order
+
+3) you are working directly with the value so it is always modifing the number.
+
+4.1)string/char array/
+.2)invalid
+.3)invalid
+.4)pointer this gets the address of the array
+.5)address this gets the address
```

Comment:
```This actually returns the type int*. Recall that a[0] would be of type int and the ampersand just returns the memory address. So &a[0] would return the pointer to the address (since a pointer is basically an address, which is why all pointers are 8 bytes no matter what they're pointing to), which would be of type int*. ```